docs/tutorials/01_basics/04_strings.md

```diff
@@ -0,0 +1,254 @@
+# Strings
+
+In the previous lessons we have already made an acquaintance with a major building block
+of most programs: strings. Let's recapitulate the basic properties:
+
+A [string](https://en.wikipedia.org/wiki/String_(computer_science)) is a sequence of [Unicode](https://en.wikipedia.org/wiki/Unicode) characters encoded in [UTF-8](https://en.wikipedia.org/wiki/UTF-8).
+A string is [immutable](https://en.wikipedia.org/wiki/Immutable_object):
+If you apply a modification to a string, you actually get a new string with the
+modified content. The original string stays the same.
+
+Strings are written as literals typically enclosed in double quote characters (`"`).
+
+## Interpolation
+
+String interpolation is a convenient method for combining strings: `#{...}` inside a string literal inserts the value of the expression between the curly braces at this position of the string.
+
+```{.crystal .crystal-play}
+name = "Crystal"
+puts "Hello #{name}"
+```
+
+The expression inside an interpolation should be kept short to either a variable or a simple method call. More complex expressions reduce code readability.
+
+The value of the expression doesn't need to be a string. Any type will do and it gets converted to a string representation by calling the `#to_s` method. This method is defined for any object. Let's try with a number:
+
+```{.crystal .crystal-play}
+name = 6
+puts "Hello #{name}!"
+```
+
+!!! note
+    An alternative to interpolation is concatenation. Instead of `"Hello #{name}!"` you could write `"Hello " + name + "!"`. But that's bulkier and has some gotchas with non-string types. Interpolation is generally preferred over concatenation.
```

Comment:
Can we be specific about the gotchas? Or just suggest that interpolation is preferred?

Reply:
Yeah, I don't know. I wouldn't just want to say interpolation is preferred, without a reason. But this is also not the place to get into details.